lib/getGhostVersion/getGhostVersion.js

```diff
@@ -0,0 +1,30 @@
+const https = require("https");
+
+function getGhostVersion() {
+  return new Promise((resolve, reject) => {
+    const options = {
+      hostname: "api.github.com",
+      path: "/repos/TryGhost/Ghost/releases/latest",
```

Comment:
Hmm, this is the latest _released_ Ghost, but that doesn't mean it's what the user is targeting. What's the advantage of checking this at all?